cc_midterm/cc_midterm/uploads.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+from .models import *
+
+
+def upload_households(households_file):
+    households_rows = households_file.split('\n')
+
+    households = []
+    for i in range(1, len(households_rows)):
+        if households_rows[i] == '': # Empty row: just move on to the next one
+            continue
+
+        row_values = households_rows[i].split(',')
+
+        h = Households(
+                hshd_num=int(row_values[0].strip()),
+                l=row_values[1],
+                age_range=row_values[2],
+                marital=row_values[3],
+                income_range=row_values[4],
+                homeowner=row_values[5],
+                hshd_composition=row_values[6],
+                hh_size=row_values[7],
+                children=row_values[8]
+            )
```

Comment:
Really not a huge fan of this, considering there are pretty easy ways around it if you're not familiar, usually via `csv` or `pandas` (although `pandas` is an external lib). For example from [this stack overflow](https://stackoverflow.com/questions/21572175/convert-csv-file-to-list-of-dictionaries):

> Use `csv.DictReader`:
> ```
> import csv
> 
> with open('test.csv') as f:
>     a = [{k: int(v) for k, v in row.items()}
>         for row in csv.DictReader(f, skipinitialspace=True)]
> ```
> Will result in :
> ```
> [{'col2': 2, 'col3': 3, 'col1': 1}, {'col2': 5, 'col3': 6, 'col1': 4}]
> ```
>

Reply:
Same goes for other related areas

Reply:
Fixed in the latest commit!